cfpb/cfgov-refresh
# Global Search: Fix search hover state

In https://github.com/cfpb/cfgov-refresh/pull/5593 I added a gray hover state for the global search at tablet and mobile sizes, but this bled over to the desktop size too, which was unintended. Additionally, I noticed the global search maybe didn't ever have a hover state beyond the cursor change. 

## Changes

- Provide text color change for global search button at desktop size.

## How to test this PR

1. Pull branch, build, and hover over global search and see that it is a text color change instead of an ugly background change like on production.


## Screenshots
![search-hover](https://user-images.githubusercontent.com/704760/84188950-6d91a580-aa62-11ea-8517-b1a1463d2e39.gif)
